includes/helpers/functions/general.php

```diff
@@ -262,10 +262,6 @@ function wpbdp_get_formfield( $id ) {
 /* Fees/Payment API */
 
 function wpbdp_payments_possible() {
-    if ( ! wpbdp_get_option( 'payments-on' ) ) {
```

Comment:
Instead of only checking for gateways, we need to replace this line with a check for active plans with a price.

Here's what should happen:
Before: The setting was on => payments are on
After: There is an enabled plan with a price => payments are on